packages/front-generator/src/common/base-generator.ts

```diff
@@ -42,13 +42,22 @@ export abstract class BaseGenerator<A, M, O extends CommonGenerationOptions> ext
     super(args, options);
     this._populateOptions(this._getAvailableOptions());
     this.destinationRoot(this._getDestRoot());
+    this.modelFilePath = this._composeModelFilePath(this.options);
+
     // @ts-ignore this.env.adapter is missing in the typings
     this.env.adapter
       .promptModule.registerPrompt('autocomplete',  AutocompletePrompt);
     this.registerTransformStream(createEjsRenameTransform());
     this.registerTransformStream(createFormatTransform());
   }
 
+  protected _composeModelFilePath(options: O) : string | undefined {
+    const {model} = options;
+    if (model == null) return undefined;
+
+    return path.isAbsolute(model) ? model: path.join(process.cwd(), model);
```

Comment:
This won't work if ` --dest` option is provided (`process.cwd()` will return the destination directory, whereas we need the directory from where the command is invoked)